### PR TITLE
update to nx-10

### PR DESCRIPTION
**Pull request by ZachJW34**
## Current Behavior
`@nx-plus/vue` and `@nx-plus/docusaurus` only work for Nx 9.

## Expected Behavior
Support for Nx 10.

## Related Issue(s)
#43

### CODE REVIEW

**Comment by ZachJW34** [comment]
TODO:

- Vue + React don't mix (introduction of root babel.config.json file) when running `nx test my-vue-app`.
- `[vue-jest]: no tsconfig.json found, defaulting to default typescript options` when running `nx test my-vue-app`. vue-jest was defaulting to the root `tsconfig.json` file, but it's been renamed to `tsconfig.base.json`. Should be using the app's `tsconfig.spec.json`.
- Update documentation.

File: libs/vue/src/schematics/application/files/tsconfig.json.template [hunk 1]
```diff
@@ -1,10 +1,19 @@
...
     "allowSyntheticDefaultImports": true,
     "types": ["webpack-env", "node"]
   },
-  "include": ["**/*.ts", "**/*.tsx", "**/*.vue"]
+  "include": [],
+  "files": [],
+  "references": [
+    {
+      "path": "./tsconfig.app.json"
+    }<% if (unitTestRunner === 'jest') {  %>,
```
Comment (ZachJW34) [inline libs/vue/src/schematics/application/files/tsconfig.json.template:14]: Remove. This is handled by the jest schematic

**Review by BuckyMaler — COMMENTED** [review]
LGTM.

Going forward I think we should align our package versions with Nx. What do you think? If you agree, then update the version for the migration to `10.0.0`.

I'd like to merge this asap, but I want us to be able to continue to provide critical fixes for Nx 9 users. I think creating a `0.6.x` branch off of latest `master` before merging this would be helpful. That way critical fixes can be either cherry picked from `master` to `0.6.x` or merged directly to `0.6.x`, and releases can be published from that branch. Does that make sense?

**Comment by ZachJW34** [comment]
I'm cool with all of that!

**Review by BuckyMaler — APPROVED** [review]
It feels good to catch up with Nx!

**Pull request merged**